src/components/timebank/timebank-content.tsx

```diff
@@ -48,16 +51,22 @@ interface Props {
  * @param props Component properties
  */
 const TimebankContent = (props: Props) => {
+  const userProfile = useAtomValue(userProfileAtom);
   const { handleDailyEntryChange, loading } = props;
 
   const [selectedEntries, setSelectedEntries] = useState<DailyEntry[]>([]);
   const [byRange, setByRange] = useState({
     dailyEntries: false
   });
   const personTotalTime = useAtomValue(personTotalTimeAtom);
+  const persons = useAtomValue(personsAtom);
   const [timespan, setTimespan] = useAtom(timespanAtom);
   const personDailyEntry = useAtomValue(personDailyEntryAtom);
   const dailyEntries = useAtomValue(dailyEntriesAtom);
+  const adminMode = UserRoleUtils.adminMode();
+  const [selectedEmployee, setSelectedEmployee] = useState<number | null>(
```

Comment:
I may not be necessary to define types here. They can be inferred.

Reply:
Removed types.